src/renderer/src/elements/pocket.ts

```diff
@@ -157,40 +142,28 @@ export class MossPocket extends LitElement {
           }
         }}
       >
+        <div slot="label" style="display:flex;align-items:center;">
+            <img src="pocket_black.png" style="height: 30px; margin-right: 10px;">
+                ${msg('Pocket:')}
+            <div style="display:inline-block;font-size:80%;color: var(--sl-color-secondary-950)">
+              <span style="background: #e0e0e0; padding: 2px 5px; border-radius: 4px; color: black;"
+                        >Alt + P</span>
+              ${msg('to open')}
+            </div>
+          </div>
+        </div>
+        <sl-icon-button slot="header-actions"
```

Comment:
A tooltip would be nice here to understand the consequences of clicking it.

Reply:
Interesting, I couldn't get sl-tooltip to work.  I had to use `title=""`..